TKeazirian.HTTPServer.Tests/RouterTests.cs

```diff
@@ -0,0 +1,50 @@
+using Xunit;
+
+namespace TKeazirian.HTTPServer.Tests;
+
+public class RouterTests
+{
+    private const string NewLine = "\r\n";
+
+    [Fact]
+    public void HandleRequestReturnsEchoBody()
+    {
+        Router router = new Router();
+        string testRequest = $"POST /echo_body HTTP/1.1{NewLine}" +
+                             $"Content-Type: text/plain{NewLine}" +
+                             $"Host: localhost:5000{NewLine}" +
+                             $"Content-Length: 11{NewLine}{NewLine}" +
+                             "Hello, World";
+
+        string expectedReturn =
+            $"HTTP/1.1 200 OK{NewLine}" +
+            $"Content-Type: plain/text{NewLine}" +
+            $"Content-Length:12{NewLine}{NewLine}" +
+            $"Hello, World";
+
+        string actualReturn = router.HandleRequest(testRequest);
+
+        Assert.Equal(expectedReturn, actualReturn);
+    }
+
+    [Theory]
+    [InlineData("/")]
+    [InlineData("/echo_test")]
+    public void HandleRequestReturns404(object value)
```

Comment:
I like that the two paths you provide are not explicitly setup in anyway. Would this still work as expected if you added these paths?

Reply:
No, it wouldn't.  I could make them more specific like `/this_path_does_not_exist` or a path I know I will not be implementing?  Or is this more of just a call-out rather than needing to change anything?